client/src/App.js

```diff
@@ -1,18 +1,21 @@
 import React, { useContext, useEffect } from 'react';
-import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
```

Comment:
We are going to host in Heroku. We do not need to use the HashRouter in this case. Let's keep the BrowserRouter as it rends better.